app/data/models/Card.py

```diff
@@ -0,0 +1,16 @@
+class Card:
+
+    def __init__(self, card_id, deck_id, word, translation ):
+        self.card_id = card_id
+        self.deck_id = deck_id
+        self.word = word
+        self.translation = translation
+
+
+    @classmethod
+    def new_card(cls, deck_id, word, translation):
```

Comment:
Im not sure what that does, but if it is creating new card and it is part of Card object then just name it create

Reply:
I have two methods new_card and card. I can't overload the constructor in python so I created those methods to be able to create a  new card where I don't have card.id yet so I can't pass it as an argument and user where I can put arguments into constructor from the database.